### PR TITLE
Details frame added. UI improvements

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/resultTable/DetailsFrame.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view.resultTable;
+
+import javax.swing.JFrame;
+import java.awt.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/resultTable/DetailsFrame.java]: Not import complet packages, only needed classes.

File: src/main/java/com/fundation/logic/view/resultTable/PopupMenu.java [hunk 2]
```diff
@@ -62,5 +63,19 @@ public void mousePressed(MouseEvent e) {
         } catch (Exception e) {
             e.getMessage();
         }
+
+        try {
+            detailsItem.addMouseListener(new MouseAdapter() {
+                @Override
+                public void mousePressed(MouseEvent e) {
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/resultTable/PopupMenu.java]: Variable name not consistent with standars, please use a variable name more descriptive.

File: src/main/java/com/fundation/logic/view/resultTable/DetailsFrame.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/resultTable/DetailsFrame.java]: done

File: src/main/java/com/fundation/logic/view/resultTable/PopupMenu.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/resultTable/PopupMenu.java]: done

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 3]
```diff
@@ -50,17 +51,17 @@
      * Initializes a MainTabs instance for the search options
      */
     public GeneralSearchPanel() {
-        Color newColor = new Color(81, 209, 246);
-        setBackground(newColor);
+        //Color newColor = new Color(81, 209, 246);
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java]: deleted this line

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 4]
```diff
@@ -50,17 +51,17 @@
...
+        //setBackground(newColor);
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java]: deleted this line

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 5]
```diff
@@ -69,8 +70,8 @@ public GeneralSearchPanel() {
         add(lblDateCreation, gbc_lblDateCreation);
 
         textFieldFromDateCreation = new JDateChooser();
-        JTextFieldDateEditor editorCreationDateFrom = (JTextFieldDateEditor) textFieldFromDateCreation.getDateEditor();
-        editorCreationDateFrom.setEditable(false);
+        //JTextFieldDateEditor editorCreationDateFrom = (JTextFieldDateEditor) textFieldFromDateCreation.getDateEditor();
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java]: deleted this line

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java [hunk 6]
```diff
@@ -69,8 +70,8 @@ public GeneralSearchPanel() {
...
+        //editorCreationDateFrom.setEditable(false);
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java]: deleted this line

File: src/main/java/com/fundation/logic/view/resultTable/DetailsTable.java [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+     */
+    public void initSetting() {
+        model = new DefaultTableModel(new Object[]{"Metadata", "MetadataInfo"}, 0) {
+            @Override
+            public boolean isCellEditable(int row, int column) {
+                return false;
+            }
+        };
+        this.setModel(model);
+        //showMetadataTable();
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/resultTable/DetailsTable.java:43]: deleted this line

File: src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/search/GeneralSearchPanel.java]: done

**Pull request merged**